packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -117,12 +118,42 @@ export class ARRenderer extends EventDispatcher<
   private yawDamper = new Damper();
   private scaleDamper = new Damper();
 
+  private controller1: any;
+  private controller2: any;
+  private controllerGrip1: any;
+  private controllerGrip2: any;
+  private intersected: any[] = [];
```

Comment:
Are these new Three.js APIs? Otherwise we should have types for them.

Reply:
Ok, I'll try adding types everywhere again.